backend/src/main/resources/application.yml

```diff
@@ -8,4 +8,6 @@ ai.versum:
     baseUrl: http://localhost:11434
   openai:
     enabled: true
-    apiKey: sk_test_1234567890
\ No newline at end of file
+    apiKey: sk_test_1234567890
+  genai:
+    GOOGLE_API_KEY: AIzaSyAWmrSGdd6g675b3d5Ptm3cKsLpOARNVr0
```

Comment:
Please provide only fake api key here, this is publicly visible and someone can use this for its own purpose. For security reasons please invalidate it on: https://aistudio.google.com/app/apikey